config/standard/forms/app/delivery.xml

```diff
@@ -1402,10 +1423,11 @@ ID: <output value=" /delivery/group_summary/r_patient_id "/></value></text>
           <geolocation tag="hidden"/>
           <patient_age_in_years tag="hidden">0</patient_age_in_years>
           <patient_contact_phone tag="hidden"/>
+          <still_pregnant tag="hidden"/>
           <group_chw_info tag="hidden">
             <chw_information/>
             <call_button/>
-            <still_pregnant>no</still_pregnant>
+            <q_still_pregnant/>
```

Comment:
I was thinking someone has a postgres view or dashboard that requires this value.

Reply:
Hmm, are people actually just consuming these standard forms right in their deployments? (Or are you thinking of test deployments?)

Either way, what if I just move my `still_pregnant` calculate into the `group_chw_info` group (so it is in the same place as the original question).  Then the report fields should still be exactly the same (with the addition of an extra `q_still_pregnant` field)?

Reply:
So, it turns out I realized why I moved the `still_pregnant` calculate out of the `group_chw_info` group in the first place. That was because the relevance condition on the `group_chw_info` group is what is causing the problem in the first place and making it so that the `still_pregnant` value is cleared (causing the other dependent fields in the form to end up being wrong.  (e.g. the `group_delivery_summary` group is only relevant if `still_pregnant = no` so if `group_chw_info` is not relevant, then the default value of `no` will be cleared from `still_pregnant` and `group_delivery_summary` will end up not being relevant either.

I have fixed things up here by putting the `still_pregnant` back where it was and then updating all the references in the form logic to just be `coalesce(${still_pregnant}, 'no')`.  It is annoying, but I think it is the best way to maintain the backwards compatibility.